cypress/integration/common/steps.js

```diff
@@ -1,8 +1,19 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+import find from 'lodash/find'
 
 const baseUrl = 'http://localhost:3000'
 const username = 'Peter Lustig'
 
+const locales = require('../../../locales')
+// TODO: use the native locale files
```

Comment:
It's alright to have redundancy in tests.

Reply:
But I does not make sense to repeat stuff that might change so you have to maintain Tests in cases where it’s not needed. At least in my opinion.

Reply:
Yes, but redundancy in tests is not soo terrible as in the implementation. Since the tests get executed on each `git push` you will have the redundancy under your radar. But not the redundancy in your implementation if it's untested.